src/lib.rs

```diff
@@ -1469,4 +1474,15 @@ mod tests {
     test_ivc_base_with::<G1, G2>();
     test_ivc_base_with::<bn256::Point, grumpkin::Point>();
   }
+
+  // SAVE_GENERATED_KEYS_TO_JSON=true cargo +nightly test test_ivc_nontrivial_with_compression_pasta --release -- --nocapture
+  #[test]
+  fn test_ivc_nontrivial_with_compression_pasta() {
+    //get boolean args from commandline to generate keys to json
+    let generate_keys_to_json = std::env::var("SAVE_GENERATED_KEYS_TO_JSON").unwrap_or_default() == "true";
```

Comment:
I would not use the separate ENVVAR for it - the intention of this test is just generation of JSONs, and it is not necessary to run it on CI, that is why you can use `test_ivc_nontrivial_with_compression_with::<G1, G2>(true);` explicitly and make this test as `ignored` (in your script you can easily run the ignored tests via `cargo test -- --ignored`)

Reply:
Good notice will change